### PR TITLE
Fix unclosed file error

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

File: plone/namedfile/scaling.py [hunk 1]
```diff
@@ -271,6 +271,12 @@ def __call__(
             data, contentType=mimetype, filename=orig_value.filename,
         )
         value.fieldname = fieldname
+
+        # make sure the file is closed to avoid error:
+        # ZODB-5.5.1-py3.7.egg/ZODB/blob.py:339: ResourceWarning:
+        # unclosed file <_io.FileIO ... mode='rb' closefd=True>
+        orig_data.close()
```
Comment (jensens) [inline plone/namedfile/scaling.py:278]: Is this for sure an open file?  In the except above https://github.com/plone/plone.namedfile/blob/7c405305d47a4a7a3ca440093b36df5e7a04fa3c/plone/namedfile/scaling.py#L223 the ``orig_data`` might be something else.

Comment (tisto) [reply plone/namedfile/scaling.py:278]: I am still investigating. Just wanted to share my current finding.  Yeah it seems to be a BlobFile:

````
> /Users/timo/workspace/plone/plone.restapi/extras/plone.namedfile/plone/namedfile/scaling.py(279)__call__()
-> orig_data.close()
(Pdb) orig_data
<_io.FileIO name='/var/folders/zw/wqv1f2gd4zdc43t1j834x5580000gn/T/BUCzqi_kw2j' mode='rb' closefd=True>
(Pdb) type(orig_data)
<class 'ZODB.blob.BlobFile'>
````

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

File: plone/namedfile/scaling.py (hunk 1 above)
Comment (tisto) [reply plone/namedfile/scaling.py]: @jensens I amended the code to only close the file if it is a BlobFile. I don't claim to fully understand everything that happens there and I am happy if someone with a deeper knowledge of the plone.namedfile code would refactor this. Pls review again. :)

**Pull request merged**